java/ql/test/library-tests/dataflow/partial/test.ql

```diff
@@ -0,0 +1,17 @@
+import java
```

Comment:
I suggest importing `PartialPathGraph` and making this a `path-problem`, to also test the flow graph.